src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IncentiveReviewsResourceTest.kt

```diff
@@ -1,15 +1,61 @@
 package uk.gov.justice.digital.hmpps.incentivesapi.resource
 
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.AfterEach
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Nested
 import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
 import uk.gov.justice.digital.hmpps.incentivesapi.integration.SqsIntegrationTestBase
+import uk.gov.justice.digital.hmpps.incentivesapi.jpa.PrisonerIepLevel
+import uk.gov.justice.digital.hmpps.incentivesapi.jpa.ReviewType
+import uk.gov.justice.digital.hmpps.incentivesapi.jpa.repository.PrisonerIepLevelRepository
+import java.time.LocalDateTime
 
 class IncentiveReviewsResourceTest : SqsIntegrationTestBase() {
+  @Autowired
+  private lateinit var repository: PrisonerIepLevelRepository
+
   @BeforeEach
-  internal fun setUp() {
+  internal fun setUp(): Unit = runBlocking {
     offenderSearchMockServer.resetAll()
     prisonApiMockServer.resetAll()
+
+    repository.deleteAll()
+    persistPrisonerIepLevel(
+      bookingId = 110001,
+      prisonerNumber = "A1409AE",
+    )
+
+    persistPrisonerIepLevel(
+      bookingId = 110002,
+      prisonerNumber = "G6123VU",
+    )
+  }
+
+  private val iepTime: LocalDateTime = LocalDateTime.now()
```

Comment:
doesn't matter hugely (or now), but would probs be nice to force the system clock "now" during these tests so that one could see all date asserts explicitly as dates.

Reply:
IMHO it doesn't matter when the time is not used or used only to check relative time differences like in this case (the assertion is on the +1 year difference)

In fact I personally find `.now()`/`.now().plusSomething()` much easier on the eye than hardcoding all the dates (which is manual and error prone, e.g. copy and paste and it's possible to end up with an older review with a date/time which is not quite right).

Just my 2 eurocents :P